bp_top/test/tb/bp_tethered/Makefile.vcs

```diff
@@ -22,9 +32,16 @@ $(LINT_DIR)/flist.vcs:
 
 LINT_COLLATERAL = $(addprefix $(LINT_DIR)/, flist.vcs wrapper.sv testbench.sv test_bp.sv bsg_tag_boot_rom.v)
 
-$(BUILD_DIR)/testbench.sv $(BUILD_DIR)/wrapper.sv $(BUILD_DIR)/test_bp.sv:
+$(BUILD_DIR)/testbench.sv $(BUILD_DIR)/test_bp.sv:
 	@sed "s/BP_CFG_FLOWVAR/$(CFG)/g" $(TB_PATH)/$(TB)/$(@F) > $@
 
+$(BUILD_DIR)/wrapper.sv:
+ifeq ($(SIM_SYNTH_VIVADO_P), 0)
+	@sed "s/BP_CFG_FLOWVAR/$(CFG)/g" $(TB_PATH)/$(TB)/$(@F) > $@
+else
+	@mv $(SYN_PATH)/results/vivado/bp_tethered.$(CFG).$(TAG).build/wrapper_synth.sv $@
```

Comment:
cp?

Reply:
Otherwise running this multiple times in a row will fail

Reply:
I thought I should make it cp. Will change it